src/Bootstrap/LoadEnvironmentVariables.php

```diff
@@ -27,10 +29,10 @@ class LoadEnvironmentVariables
      * Create a new loads environment variables instance.
      *
      * @param  string  $path
-     * @param  string|null  $name
+     * @param  string  $name
      * @return void
      */
-    public function __construct($path, $name = null)
+    public function __construct($path, $name = '.env')
```

Comment:
This is a breaking change.

Reply:
How so? `null` becomes `.env` upstream.

https://github.com/vlucas/phpdotenv/blob/master/src/Store/StoreBuilder.php#L98